Components/image/image.css

```diff
@@ -0,0 +1,16 @@
+
+.responsive{
+  max-width: 100%;
+  height: auto;
```

Comment:
instead of writing auto try using object-fit : contain / cover, will be better for responsivness